src/Components/Footer.js

```diff
@@ -0,0 +1,32 @@
+import * as React from "react";
+import { styled } from "@mui/material/styles";
+import Box from "@mui/material/Box";
+import Paper from "@mui/material/Paper";
+import Grid from "@mui/material/Grid";
+
+const Item = styled(Paper)(({ theme }) => ({
+  textAlign: "center",
+}));
+
+export default function StickyFooter() {
+  return (
+    <Box className="footer" sx={{ flexGrow: 1 }}>
+      <Grid container spacing={3}>
+        <Grid item xs>
+          <Item id="footer"><a href="#">Sobre Nosotros</a></Item>
+          <Item id="footer"><a href="#">Terminos y condiciones</a></Item>
```

Comment:
los links en react creo que funcionan mejor con el elemento <Link to=""> de react router

Reply:
con <Link>

Reply:
Como que no te aparece? <Link to="">

Reply:
pero seria con ese si, con el <Link to=""

Reply:
Tengo que actualizar esta parte del código, no tenia idea del router en ese momento